Dockerfile.arm

```diff
@@ -0,0 +1,13 @@
+FROM resin/armel-debian:jessie
```

Comment:
`GOARM=5` == `armel`, `GOARM=7`== `armhf`, `GOARCH=arm64` == `aarch64/arm64`
Raspberry Pi 1: `GOARM=6`, **`armhf`** (Other `GOARM=6` boards doesn't have `armhf` support)
Raspberry Pi 2: `GOARM=7`, `armhf`
Raspberry Pi 3, Estuary: `GOARCH=arm64`
`GOARCH=arm64` may run `GOARM=7` binaries, `GOARM=7` may run `GOARM=6` and `GOARM=5` binaries, but not the other way around.
The thing is that often `GOARM=7` and `armhf` is considered equal, but it's not.

All `armhf/*` images on Docker Hub are therefore `ARMv7` and all `armel/*` are `ARMv5`. RPi 1 is between those,  and `resin/rpi-raspbian` is `GOARM=6` and `armhf`

OK, that was hard to try to explain. For maximum support though, go with official `armel/debian`, as there is no significant perfomance boost in having `armhf/debian` and dropping support for some boards.